.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+*Replace this paragraph with a description of what this PR is changing or adding, and why. Consider including before/after screenshots.*
+
+*List which issues are fixed by this PR. For larger changes, raising an issue first helps
+reduce redundant work.*
+
+## Pre-launch Checklist
+
+- [ ] I read the [Flutter Style Guide] _recently_, and have followed its advice.
+- [ ] I signed the [CLA].
```

Comment:
This should have a checkbox for reading the contributor's guide in this repo: https://github.com/flutter/samples/blob/master/CONTRIBUTING.md.

That file needs an update pretty badly. I'll assign myself an issue to do it.